server/src/main/java/keywhiz/ServiceModule.java

```diff
@@ -75,6 +77,12 @@ public ServiceModule(KeywhizConfig config, Environment environment) {
     bind(KeywhizConfig.class).toInstance(config);
   }
 
+  // AuditLog
```

Comment:
I haven't worked much with injection before; if this should be added somewhere else, let me know.